modules/hcp-ibm-power-heterogeneous-nodepools-agent-hc-dns.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ibm-power-heterogeneous-nodepools-agent-hc-dns_{context}"]
+= DNS configuration for hosted control planes
+
+Domain Name Service (DNS) configuration for hosted control planes is done basically for external clients to reach ingress controllers so that it can route traffic to internal components. Configuring this setting ensures that traffic is routed to either worker node e.g. `ppc64le` or `x86_64` node.
```

Comment:
```suggestion
Domain Name Service (DNS) configuration for hosted control planes enables external clients to reach ingress controllers so that they can route traffic to internal components. Configuring this setting ensures that traffic is routed to either `ppc64le` or `x86_64` compute node.
```